static/app/bootstrap/initializeSdk.tsx

```diff
@@ -25,6 +24,12 @@ const SPA_MODE_TRACE_PROPAGATION_TARGETS = [
   'sentry.dev',
 ];
 
+let lastEventId: string | undefined;
+
+export function getLastEventId(): string | undefined {
```

Comment:
Had to add this to manually keep the last event ID, as we need this for the error report in a component 🤔 cc @lforst not sure if this is an ideal solution...?

Reply:
probably fine